src/main/java/com/ausregistry/jtoolkit2/se/DomainCreateCommand.java

```diff
@@ -89,5 +89,71 @@ public DomainCreateCommand(String name, String pw, String registrantID,
                     "authInfo"),
                 "pw").setTextContent(pw);
     }
+
+    /**
+     * Most verbose constructor for a domain:create EPP command. All core EPP
+     * domain:create attributes may be set using this constructor.
+     * nameserver will be passed as attribute in this command
+     *
+     * @throws IllegalArgumentException if {@code name} or {@code pw} are {@code null}.
+     */
+    public DomainCreateCommand(String name, String pw, String registrantID,
+        String[] techContacts, String[] adminContacts,
+        String[] billingContacts, Host[] nameservers, Period period, boolean hostAttribute) {
```

Comment:
This new constructor duplicated most of the code from the original one. According to the javadoc, this is 
> Most verbose constructor for a domain:create EPP command.

Then the old constructor needs to call `this(..)`.

Besides, why do we need the last boolean parameter?

Reply:
I have added an extra parameter because there were 2 constructors which were passing null in first argument (nameservers) so again call will be ambiguous if we add another constructor with Host[] as first parameter in place of String[]. I will work on this to remove duplicacy